central/graphql/resolvers/image_components.go

```diff
@@ -56,9 +58,8 @@ type ImageComponentResolver interface {
 	ActiveState(ctx context.Context, args RawQuery) (*activeStateResolver, error)
 	DeploymentCount(ctx context.Context, args RawQuery) (int32, error)
 	Deployments(ctx context.Context, args PaginatedQuery) ([]*deploymentResolver, error)
-	FixedIn(ctx context.Context) string
 	FixedBy(ctx context.Context) string
-	ID(ctx context.Context) graphql.ID
+	Id(ctx context.Context) graphql.ID
```

Comment:
nit: `ID` because it is abbrev

Reply:
The reason I'm making this change is that the `Id` function is autogen'd based off the proto